### ISSUE TITLE
add md5 and sha1 checksum links to tools download page

### ISSUE THREAD

**Issue by martinlippert** [spring-io/sagan#726]
As described here (https://github.com/spring-projects/spring-ide/issues/144) is would be great if we could include links to the md5 and sha1 checksums of the downloads again on the tools download page: https://spring.io/tools/sts/all

A design sketch is included in the issue mentioned above.

**Comment by davidnewcomb** [comment]
This is the second time your main STS download has been corrupted and there has been no MD5 hashes to check against.
I'm not going to bother reporting it any more and after spending the day battling STS's proxy bugs I'm beginning to think that it's time that STS and I parted company.

![image](https://user-images.githubusercontent.com/951359/28421685-cc09fcb0-6d5d-11e7-954b-f4c0773a2211.png)

**Comment by martinlippert** [comment]
MD5 (STS 3.9.0, Windows 64bit): http://dist.springsource.com/release/STS/3.9.0.RELEASE/dist/e4.7/spring-tool-suite-3.9.0.RELEASE-e4.7.0-win32-x86_64.zip.md5

SHA1 (STS 3.9.0, Windows 64bit):
http://dist.springsource.com/release/STS/3.9.0.RELEASE/dist/e4.7/spring-tool-suite-3.9.0.RELEASE-e4.7.0-win32-x86_64.zip.sha1

I think I described how to get to the MD5 and SHA1 files for each download manually (actually just add the "md5" or ".sha1" to the regular download link), as long as we don't have them again as direct visible links on the tools download page.

The hashes are available for every release, I don't create them on demand. So feel free to grab them via those URLs anytime.

Sorry for the inconvenience.

**Comment by davidnewcomb** [comment]
You are right, you did mention it before but it's too easy to forget for all the reasons I mentioned before. The download failed the MD5 checksum, can you check it is ok at your end.

**Comment by martinlippert** [comment]
The double checked the md5 and it is the correct md5 of the download zip for Win 64bit.
